packages/core/src/associations/belongs-to-many.ts

```
@@ -1344,6 +1391,35 @@ export type BelongsToManyCreateAssociationMixin<T extends Model> = (
+ *   declare createRoles: BelongsToManyCreateAssociationsMixin<Role>;
+ * }
+ *
+ * User.belongsToMany(Role, { through: UserRole });
+ * ```
+ *
+ * @see Model.belongsToMany
+ */
+export type BelongsToManyCreateAssociationsMixin<T extends Model> = (
+  valuesArray: Array<CreationAttributes<T>>,
+  options?: BelongsToManyCreateAssociationsMixinOptions<T>,
+) => Promise<Array<T>>;
```

ESLint

[error] Array type using 'Array<T>' is forbidden for simple types. Use 'T[]' instead.

(@typescript-eslint/array-type)